backend/src/apiserver/server/experiment_server.go

```diff
@@ -44,7 +57,17 @@ func (s *ExperimentServer) ListExperiment(ctx context.Context, request *api.List
 		return nil, util.Wrap(err, "Failed to create list options")
 	}
 
-	experiments, total_size, nextPageToken, err := s.resourceManager.ListExperiments(opts)
+	filterContext, err := ValidateFilter(request.ResourceReferenceKey)
+	if err != nil {
+		return nil, util.Wrap(err, "Validating filter failed.")
+	}
+
+	refKey := filterContext.ReferenceKey
+	if refKey != nil && refKey.Type != common.Namespace {
```

Comment:
When refKey is nil, IMO backend should reject the request if in multi user mode, and default to namespace "" when not in multi user mode.
WDUT?

Reply:
Update the code to reject empty namespace in ListExperiemntsRequest under multi-user mode.